lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -7951,6 +7954,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenMatmulOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenMvOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenLinalgCrossOp>(patterns);
+    // addPatternIfTargetOpIsIllegal<DecomposeAtenKthvalueOp>(patterns);
```

Comment:
Is this intended  to be commented out? Typically we control which are enabled via the pass settings for DecomposeComplexOps.

Reply:
I accidentally left this here from when I tried to implement the Kthvalue lowering as a decomposition. It serves no purpose, so I have removed it.